app/javascript/react/components/bands/BandIndex.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import BandTile from './BandTile'
+
+const BandsIndexPage = (props) => {
+  const [bands, setBands] = useState([])
+
+  const fetchBands = async () => {
+    try{
+      const response = await fetch('/api/v1/bands')
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const parsedBandsArray = await response.json()
+      setBands(parsedBandsArray)
+    } catch(err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+  
+  useEffect(() => {
+    fetchBands()
+  }, [])
+
+  const bandTiles = bands.map((band) => {
+    return (
+      <BandTile
+        key={band.id}
+        id={band.id}
+        name={band.name}
+        biography={band.biography}
```

Comment:
If you're passing all of the properties of a `band` to the BandTile, you can pass the whole `band` object as a prop. This helps in the future if you add more properties to bands so that you don't have to come back here to pass more props
```js
<BandTile
  key={band.id}
  band={band}
/>
```